### DISCUSSION TITLE
Please help

### DISCUSSION THREAD

**Discussion by Sector15a** [community/community#136083] [New to GitHub]
![6B7591BB-E3E3-45A9-9B16-D8D803B78E90](https://github.com/user-attachments/assets/d5b7b43b-63aa-4a75-9e91-3118ce196f2e)
![E67A8E75-BBA8-4F41-8793-E50F73BA0CD3](https://github.com/user-attachments/assets/eb64ae29-eade-43d6-8e3e-8a7a480
![F2E72F7C-0FEC-452C-AFC7-AC124984D243](https://github.com/user-attachments/assets/a6bee86e-dfae-42af-bbca-e9d715b57141)
![Uploading 7F94FA91-5183-46BC-BA92-014138576D83.jpeg…
![B143D62B-C28A-4A7E-A2C3-520BE032B4E9](https://github.com/user-attachments/assets/cfdcc5f0-076d-4c35-b203-ef6d399d6599)
![DD95BCD0-D2E7-483C-A02C-20FA86ABED45](https://github.com/user-attachments/assets/23a375e1-fa5a-4968-9476-9e901a995539)
]()
efc8d)
![3715AFF8-75C3-4F2F-95D8-EB063138609D](https://github.com/user-attachments/assets/be770d2b-ccd9-482f-8643-72fa73a72899)
Someone has control over my iphone

**Comment by Sector15a** [comment]
Please help

**Comment by ghostinhershell** [comment]
Hi @Sector15a, 👋🏻 we really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
